scripts/python/add_users_everyone_github_team.py

```diff
@@ -22,31 +23,7 @@ def print_stack_trace(message):
         del exc_info
 
 
-if len(sys.argv) == 2:
-    # Get the GH Action token
-    oauth_token = sys.argv[1]
-else:
-    print("Missing a script input parameter")
-    sys.exit(1)
-
-# Setup a transport and client to interact with the GH GraphQL API
-try:
-    transport = AIOHTTPTransport(
-        url="https://api.github.com/graphql",
-        headers={"Authorization": "Bearer {}".format(oauth_token)},
-    )
-except Exception:
-    print_stack_trace("Exception: Problem with the API URL or GH Token")
-
-try:
-    client = Client(transport=transport, fetch_schema_from_transport=False)
-except Exception:
-    print_stack_trace("Exception: Problem with the Client.")
-
-github_service = GithubService(oauth_token, "moj-analytical-services")
-
-
-def organisation_team_id_query() -> gql:
+def organisation_team_id_query() -> DocumentNode:
```

Comment:
[MyPy](https://mypy.readthedocs.io/en/stable/) spotted this was the incorrect type 👀

Reply:
Yeah if mypy can detect the underlying library data type happy to go with that